package.json

```diff
@@ -12,7 +12,8 @@
     "cov": "jest --coverage",
     "start": "webpack && node build/gateway.js",
     "debug-ide": "webpack && node --debug-brk=5858 build/gateway.js",
-    "debug": "webpack && node --inspect build/gateway.js"
+    "debug": "webpack && node --inspect build/gateway.js",
+    "postinstall": "cp node_modules/speaktome-web/build/stm_web.min.js static/js/lib/"
```

Comment:
Interesting, I had a similar issue with the client side router library and was wondering what to do for dependency management for front end things.

This seems a bit brittle though if the package gets updated. @hobinjk do you know of a better way of doing this?

Reply:
I implemented a copywebpack plugin here as suggested by @hobinjk